packages/node_modules/samples/browser-call-with-screenshare/test/wdio/spec/normal-dialing.js

```diff
@@ -67,6 +67,8 @@ describe('samples', () => {
       });
 
       it('starts screensharing', () => {
+        // Pausing for peerConnection to finish negotiating
+        browser.pause(10000);
```

Comment:
Do we have a bug where if you try to start sharing too quickly, it will throw an error? If so, can you create a JIRA task for either a bugfix or a `canShare()` functionality to be added to the meeting object?